api/v1alpha1/component_types.go

```diff
@@ -165,6 +165,10 @@ type ComponentStatus struct {
 	// Example: 41fbdb124775323f58fd5ce93c70bb7d79c20650.
 	LastBuiltCommit string `json:"lastBuiltCommit,omitempty"`
 
+	// The last digest image component was promoted with.
+	// Example: 5ca85b7f7b9da18a9c4101e81ee1d9bac35ac2b0b0221908ff7389204660a262.
```

Comment:
Since the field should contain an image that's referenced via digest, I think that the example should look something like this:
```
quay.io/someorg/somerepository@sha256:5ca85b7f7b9da18a9c4101e81ee1d9bac35ac2b0b0221908ff7389204660a262
```

Reply:
got it, 
fixed